core/layout_api.php

```diff
@@ -969,15 +974,13 @@ function layout_main_content_end() {
  */
 function layout_page_content_begin() {
 	echo '  <div class="page-content">' , "\n";
-	echo '    <div class="row">' , "\n";
```

Comment:
I moved the markup to the calling function (one level up) it should not have any side effect.